gulpfile.js

```diff
@@ -0,0 +1,40 @@
+const gulp = require('gulp');
```

Comment:
It really, shouldn't be in the root. It only applies to the project that produces the js file. We're not using gulp for project builds. Move it to the typescript client then.